### PR TITLE
Loop precision scoop tabel

### PR DESCRIPTION
**Pull request by jalelmahjoubi**

### CODE REVIEW

File: src/model/transformations/collected-variable.js [hunk 1]
```diff
@@ -270,7 +286,11 @@ export function storeToRemote(store, componentsStore) {
         model.Scope = collectedScop.loop.id
       }
     }
-    
+    const dynamique = getTableDynamique(componentsStore, id);
```
Comment (EmmanuelDemey) [inline src/model/transformations/collected-variable.js:289]: You should maybe add unit test for this new transformation.

File: src/model/transformations/collected-variable.js [hunk 2]
```diff
@@ -207,6 +208,21 @@ function getCollectedScope(questionsLoop, id, componentsStore) {
   })
   return isfound;
 }
+
+function getTableDynamique(componentsStore, id) {
+  let tableId = "";
+  Object.values(componentsStore).filter(components => 
+    components.type === QUESTION && 
+    components.responseFormat.type === "TABLE" &&
```
Comment (EmmanuelDemey) [inline src/model/transformations/collected-variable.js:216]: We have constants somewhere for the TABLE and LIST strings

Comment (jalelmahjoubi) [reply src/model/transformations/collected-variable.js]: @EmmanuelDemey  je viens de faire un Push

**Comment by EmmanuelDemey** [comment]
Please do a rebase from the acceptance branch in order to remove the conflicting files.

File: src/layout/generic-input/utils/generic-input-utils.js [hunk 3]
```diff
@@ -213,6 +213,7 @@ export function getNewQuestionPlaceholder(components, activeComponent) {
   let heavySequenceId;
   let heavySubsequenceId;
   let heavyQuestionId;
+  console.log("activeComponent", activeComponent)
```
Comment (EmmanuelDemey) [inline src/layout/generic-input/utils/generic-input-utils.js:216]: You should remove this console.log

**Pull request merged**